### ISSUE TITLE
Image does not display with status on discord

### ISSUE THREAD

**Issue by ghost** [Pizzabelly/EasyRP#123]
Hello all the code works except for the image showing for discord would love some help on that

**Comment by BadMaliciousPyScripts** [comment]
Do you have uploaded your images as assets to the Rich Presence app? If no then do so. You can give this images a name then and use this name in the config file.
Example:
LargeImage=Picture1
![Annotation 2020-10-06 160412](https://user-images.githubusercontent.com/64361203/95212397-f1c81f80-07ed-11eb-8ab1-7d3672f64c6f.png)
Just press the button and choose a image.

**Comment by StockierAsp9634** [comment]
Have done this, the issue continues to persist.

**Comment by bhavyagosai** [comment]
Sometimes it takes a while to show it on the RP. Have you waited enough?

**Comment by StockierAsp9634** [comment]
Yeah, I ended up fixing it by being patient. Thanks for the advice!

**Comment by StockierAsp9634** [comment]
Looks like the Discord Developer Portal has an issue where it does not immediately save art assets uploaded to the Rich Presence Tab. After coming back to the website, the issue is gone.
